packages/datagateway-search/src/table/datasetSearchTable.component.tsx

```diff
@@ -17,17 +22,16 @@ import {
   useDateFilter,
   useIds,
   useLuceneSearch,
-  useSort,
   useRemoveFromCart,
+  useSort,
   useTextFilter,
-  DatasetDetailsPanel,
-  ISISDatasetDetailsPanel,
-  DLSDatasetDetailsPanel,
 } from 'datagateway-common';
-import { TableCellProps, IndexRange } from 'react-virtualized';
+import { isLandingPageSupportedForHierarchy } from 'datagateway-common/src';
```

Comment:
This line is what's causing the build failure 😅 we're importing this from `datagateway-common/src` and so importing a typescript file directly instead of the transpiled JS file!